string name
uint8 bus_num
uint8 device_num
uint32 speed
uint8 mode
---
SPIInstance inst
bool result